pennylane/measurements/mid_measure.py

```diff
@@ -74,7 +74,7 @@ def func():
     >>> func()
     tensor([1., 0.], requires_grad=True)
 
-    Mid circuit measurements can be manipulated using the following dunder methods
+    Mid circuit measurements can be manipulated using the following dunder methods:
```

Comment:
agreed. if they are so curious, the note below will help them.
```suggestion
    Mid circuit measurements can be manipulated using the following arithmetic operators:
```